app/lib/services/eligibility/targeted_delivery_funding.rb

```diff
@@ -18,6 +19,8 @@ def call
         return false if institution.is_a?(PrivateChildcareProvider)
         return false if institution.number_of_pupils.nil?
         return false if institution.number_of_pupils.zero?
+
+        return false if employment_role == "lead_mentor_for_accredited_itt_provider"
```

Comment:
Not a blocker on merging.

This string is being repeated in various places around the app and for checking. It'd be nice to have this moved into a constant somewhere, perhaps on Forms::YourEmployment and then referred to elsewhere via that constant. To guard against typos and changes in the future.

Reply:
agreed 👍